platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/iss/IssDetector.java

```diff
@@ -167,54 +154,59 @@ public ConsensusHashManager(
         if (ignoredRound != DO_NOT_IGNORE_ROUNDS) {
             logger.warn(STARTUP.getMarker(), "No ISS detection will be performed for round {}", ignoredRound);
         }
+        this.issMetrics = new IssMetrics(platformContext.getMetrics(), addressBook);
     }
 
     /**
      * This method is called once all preconsensus events have been replayed.
      */
-    public void signalEndOfPreconsensusReplay() {
+    public void signalEndOfPreconsensusReplay(@Nullable final Object ignored) {
         replayingPreconsensusStream = false;
     }
```

Comment:
I'm assuming the argument is so that we can call this using a wire? I'm wondering if it might be cleaner to remove this argument, and then do something like this at bind time: `.bind(x -> issDetector.signalEndOfPreconsensusReplay())`;

Reply:
I have seen this pattern elsewhere and so I reused it. The framework should definitely support this paradigm, but it should be done in a separate PR